crates/abq_runners/generic_test_runner/src/lib.rs

```diff
@@ -1425,7 +1425,7 @@ pub fn execute_wrapped_runner(
 
                     *manifest_message = Some(real_manifest.clone());
                     let init_meta = real_manifest.manifest.init_meta.clone();
-                    let flattened = Manifest::flatten(real_manifest.manifest.members);
+                    let flattened = Manifest::flatten_manifest(real_manifest.manifest.members);
```

Comment:
@ayazhafiz this is the only place outside of queue that flatten is used (and the thing preventing me from pulling `flatten_manifest` out of `abq_utils`